tests/docker/universum_test_env/Dockerfile

```diff
@@ -1,23 +1,39 @@
-FROM ubuntu:bionic
+FROM ubuntu:bionic AS universum_test_env_no_vcs
 ARG PYTHON=python3.7
 
+# Please note: apt-get install will produce the following message in stderr:
+# 'debconf: delaying package configuration, since apt-utils is not installed`
+# In scope of non-interactive configuration there's no need to fix it
+
 # Update package list and install wget
-RUN apt update && apt install -y wget software-properties-common
+RUN apt-get update && apt-get install -y wget software-properties-common
 
 # Install latest python & pip
-RUN add-apt-repository ppa:deadsnakes/ppa && apt update
-RUN DEBIAN_FRONTEND=noninteractive apt install -y ${PYTHON}-dev ${PYTHON}-distutils gnupg2 libssl-dev build-essential
-RUN wget --no-check-certificate -O get-pip.py 'https://bootstrap.pypa.io/get-pip.py'
+RUN add-apt-repository ppa:deadsnakes/ppa && apt-get update
+RUN apt-get install -y ${PYTHON}-dev ${PYTHON}-distutils gnupg2 libssl-dev build-essential
+
+# Please note: wget is writing logs to stderr, these logs are not any kind of warning
+RUN wget --no-verbose --no-check-certificate -O get-pip.py 'https://bootstrap.pypa.io/get-pip.py'
 RUN ${PYTHON} get-pip.py
 
+FROM universum_test_env_no_vcs AS universum_test_env_no_p4
+ARG PYTHON=python3.7
```

Comment:
Do you need to repeat this line in every stage?

Reply:
If I don't, the next stage assumes there's no such argument. Not sure if there's a better way to deal with it